Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/SyncPerformanceContext.swift

```diff
@@ -0,0 +1,307 @@
+// swiftlint:disable force_try
+import MongoSwift
+import StitchCore
+import StitchCoreSDK
+import StitchCoreAdminClient
+import StitchDarwinCoreTestUtils
+@testable import StitchCoreRemoteMongoDBService
+import StitchCoreLocalMongoDBService
+@testable import StitchRemoteMongoDBService
+
+class SyncPerformanceContext {
+    var client: StitchAppClient?
+    var mongoClient: RemoteMongoClient?
+    var coll: RemoteMongoCollection<Document>?
+
+    var dbName = ""
+    var collName = ""
+    var userId = ""
+
+    let joiner = ThrowingCallbackJoiner()
+    let streamJoiner = StreamJoiner()
+    let testParams: TestParams?
+    let transport: FoundationInstrumentedHTTPTransport?
+    let harness: SyncPerformanceIntTestHarness?
```

Comment:
[minor] none of the above lets should be optional. If you feel they must, then this was likely designed wrong